pegasus/sites.v3/code.org/views/plc_csd_nav.haml

```diff
@@ -1,15 +1,15 @@
 #about_nav
   %span.about_list
-    -current_url = '/educate/professional-learning/cs-discoveries'
+    -target_url = '/educate/professional-learning/cs-discoveries'
     -{'Middle School Courses'=>'/educate/curriculum/middle-school',
       'About the Professional Learning Program'=>'#about',
       'How does it work?'=>'#components',
       'What are the commitments?'=>'#commitments',
       'Who can participate?'=>'#participate',
       'Where is it offered?'=>'#locations',
       'How to apply?'=>'#apply'}.each_pair do |label,url|
```

Comment:
What about `base_url` and `target_url`? Because I don't really see the first link being an `additional_url`?